themes/sensei-course-theme/templates/default/lesson.html

```diff
@@ -1,64 +1,63 @@
-<!-- wp:sensei-lms/ui {"className":"sensei-version--4-8-0", "elementClass": "sensei-course-theme__header"} -->
-<div class="wp-block-sensei-lms-ui sensei-course-theme__frame sensei-course-theme__header sensei-version--4-8-0"><!-- wp:group {"style":{"spacing":{"padding":{"right":"24px","left":"24px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
-    <div class="wp-block-group" style="padding-right:24px;padding-left:24px"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-        <div class="wp-block-group"><!-- wp:site-logo {"width":50} /-->
+<!-- wp:sensei-lms/ui {"elementClass":"sensei-course-theme__header","className":"sensei-version--4-8-0"} -->
+<div class="wp-block-sensei-lms-ui sensei-course-theme__frame sensei-version--4-8-0 sensei-course-theme__header">
+	<!-- wp:group {"style":{"spacing":{"padding":{"right":"24px","left":"24px","top":"6px","bottom":"6px"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group" style="padding-top:6px;padding-right:24px;padding-bottom:6px;padding-left:24px">
+		<!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+		<div class="wp-block-group"><!-- wp:site-logo {"width":50,"style":{"color":{"duotone":["#1a4548","#ffe2c7"]}}} /-->
```

Comment:
Was the duotone filter intentional?

Reply:
No its not, great catch :) Removed it in 7f7b283541ee0e3ba29a203e62d69a84470e0367